Core/RoverControllerPresets/DriveControllers/EricSOnController.cs

```diff
@@ -34,9 +36,8 @@ public RoverControl CalculateMoveVector()
 		if (!Mathf.IsEqualApprox(forcedSteer, 0f, 0.05f))
 			vec.Y = forcedSteer / 4f;
 
-		if (Input.IsActionPressed("camera_zoom_mod"))
-			vec /= 8f;
+		Vector3 vector = new Vector3(vec.X, vec.Y, 0);
 
-		return RoverControlVec2Extension.FromVector2(vec);
+		return RoverControlVec3Extension.ToRoverControl(vector);
```

Comment:
Prefer using extension methods from extended classes.
```suggestion
		return vector.ToRoverControl();
```
see:
https://github.com/wisniax/RoverControlApp/blob/3b3e3cfab920852433be6b94e847b16dd4c70398/Core/RoverControllerPresets/MqttClasses.RoverControlVec3Extension.cs#L12-L13

Reply:
Adressed in [5ed280b](https://github.com/wisniax/RoverControlApp/pull/36/commits/5ed280b9da4555ee40b428e3d684a0ecffc34549)